plugins/polio/js/src/constants/translations/en.json

```diff
@@ -45,6 +45,8 @@
     "iaso.polio.form.label.agNopvGroup": "AG/nOPV Group",
     "iaso.polio.form.label.approval": "Approval",
     "iaso.polio.form.label.approval_confirmed": "Approval confirmed",
+    "iaso.polio.form.label.approve_unicef": "Approve (UNICEF)",
+    "iaso.polio.form.label.approve_who": "Approve (WHO)",
```

Comment:
Same here(This key is not translated in french(fr.json))

Reply:
I did it